### PR TITLE
Modify unicode tests file name

### PR DESCRIPTION
**Pull request by ningyougang**
The special character `:` of filename is not valid in windows.
So it is necessary to remove the special character.

But i am not sure below problem. For example:
the file name: nodejs:6.txt, whether used by other programme.
Even though I searched by the full name of the file, the file name is not used by other programme,
but the programme may splice the filename dynamically.

## Related issue and scope

- [ ] I opened an issue to propose and discuss this change (#????)

## My changes affect the following components

- [ ] API
- [ ] Controller
- [ ] Message Bus (e.g., Kafka)
- [ ] Loadbalancer
- [ ] Invoker
- [ ] Intrinsic actions (e.g., sequences, conductors)
- [ ] Data stores (e.g., CouchDB)
- [ ] Tests
- [ ] Deployment
- [ ] CLI
- [ ] General tooling
- [x] Documentation

## Types of changes

- [ ] Bug fix (generally a non-breaking change which closes an issue).
- [ ] Enhancement or new feature (adds new functionality).
- [ ] Breaking change (a bug fix or enhancement which changes existing behavior).

## Checklist:

- [x] I signed an [Apache CLA](https://github.com/apache/incubator-openwhisk/blob/master/CONTRIBUTING.md).
- [ ] I reviewed the [style guides](https://github.com/apache/incubator-openwhisk/wiki/Contributing:-Git-guidelines#code-readiness) and followed the recommendations (Travis CI will check :).
- [ ] I added tests to cover my changes.
- [ ] My changes require further changes to the documentation.
- [ ] I updated the documentation where necessary.

### CODE REVIEW

**Comment by ningyougang** [comment]
I am not sure below problem. for example:
the file name: nodejs:6.txt, whether used by other programme.
Even though I searched by the full name of the file, the file name is not used by other programme,
but the programme may splice the filename dynamically.

**Comment by rabbah** [comment]
Ah windows! Thank for that.
You'll need to adjust the test here https://github.com/apache/incubator-openwhisk/blob/5b13c4c90e4f8d59156c862c2ffe57ef027e5aee/tests/src/test/scala/system/basic/WskUnicodeTests.scala#L59

And the docs/actions-new.md.

**Comment by ningyougang** [comment]
@rabbah ,
I modified the `WskUnicodeTests.scala`
It seems has no need to modify docs/actions-new.md

File: docs/actions-new.md [hunk 1]
```diff
@@ -86,7 +86,13 @@ The standard test action is shown below in JavaScript. It should be adapted for
 new language and added to the [test artifacts directory](../tests/dat/actions/unicode.tests)
 with the name `<runtime-kind>.txt` for plain text file or `<runtime-kind>.bin` for a
 a binary file. The `<runtime-kind>` must match the value used for `kind` in the corresponding
-runtime manifest entry.
```
Comment (rabbah) [inline docs/actions-new.md]: how about this:

The `<runtime-kind>` must match the value used for `kind` in the corresponding
runtime manifest entry,  replacing `:` in the kind with a `-`. For example, a plain text function for`nodejs:8` becomes `nodejs-8.txt`.

Comment (ningyougang) [reply docs/actions-new.md]: Oh. good, change `:` to `-` in `<runtime-kind>.txt` or `<runtime-kind>.bin`
Already modified.

**Comment by ningyougang** [comment]
Can merge this pr as soon as possible? because in my local, after rebasing the upsteam codes, report below
```
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)
        deleted:    tests/dat/actions/unicode.tests/nodejs:6.txt
        deleted:    tests/dat/actions/unicode.tests/nodejs:8.txt
        deleted:    tests/dat/actions/unicode.tests/php:7.1.txt
        deleted:    tests/dat/actions/unicode.tests/php:7.2.txt
        deleted:    tests/dat/actions/unicode.tests/python:2.txt
        deleted:    tests/dat/actions/unicode.tests/python:3.txt
        deleted:    tests/dat/actions/unicode.tests/swift:3.1.1.txt
        deleted:    tests/dat/actions/unicode.tests/swift:4.1.txt
Untracked files:
  (use "git add <file>..." to include in what will be committed)
        tests/dat/actions/unicode.tests/nodejs
        tests/dat/actions/unicode.tests/php
        tests/dat/actions/unicode.tests/python
        tests/dat/actions/unicode.tests/swift
no changes added to commit (use "git add" and/or "git commit -a")
```

**Pull request merged**